.github/workflows/devnet-deploy.yml

```diff
@@ -127,11 +133,20 @@ jobs:
           # wait for port-forwards to establish
           sleep 5
 
-          docker run --rm --network host $AZTEC_DOCKER_IMAGE bootstrap-network \
-            --rpc-url http://127.0.0.1:$PXE_PORT \
-            --l1-rpc-url http://127.0.0.1:$ETHEREUM_PORT \
-            --l1-chain-id "$L1_CHAIN_ID" \
-            --mnemonic "$MNEMONIC" \
-            --json | tee ./basic_contracts.json
+          if ${{ inputs.sepolia_deployment }}; then
```

Comment:
Not for now, but this feels a little. A bunch of the script is to set up port forwarding, but we use none of that. Seems we should just have a different step, and condition on `inputs.sepolia_deployment`, and then a common step beneath for uploading.

Reply:
added [here](https://github.com/AztecProtocol/aztec-packages/issues/10757)